spokestack/io/sound_device.py

```diff
@@ -0,0 +1,73 @@
+"""Audio input component based on the sounddevice library."""
+from typing import Any
+
+import numpy as np
+import sounddevice as sd
+
+
+class SoundDeviceInput:
+    """
+    Audio input source using sounddevice.
+
+    Parameters
+    ----------
+    sample_rate : int, optional
+        audio sample rate, by default 16000
+    frame_width : int, optional
+        size of the audio frame, by default 20
```

Comment:
```suggestion
        size of the audio frame in milliseconds, by default 20
```
I'm used to seeing "default: <value>" or "defaults to <value>" rather than "by default", but that's a hardcore nitpick.